### ISSUE TITLE
v1.9.4-rc4 Default status list on ticket settings

### ISSUE THREAD

**Issue by quik87** [osTicket/osTicket#1403]
The select list of 'Default status' is empty all the time and not recognize states from states list.

**Comment by YantaR** [comment]
Problem exists with Polish language. there's no default status list. what i can do?

**Comment by greezybacon** [comment]
@YantaR head to Admin Panel -> Manage -> Lists -> Ticket Statuses and add two new statuses, one for open and one for closed. Then, click the [Properties] button for each after saving, and set them up appropriately.

**Comment by TinoDidriksen** [comment]
@greezybacon, that solution doesn't work. I've just installed from git (1.9.8-1-g1f5dcc4) using the Danish language pack, and Default Status was similarly empty. However, Manage -> Lists does not allow modification of list with ID 1; I can't even enter the Items tab for it.
Truncating the SQL table and re-adding a fresh list from Manage (thus making it ID 1 again) confirms this - the list is not editable. But even if I add a list ID 1 with items, or add items via MySQL, Tickets -> Default Status remains empty.

**Comment by hogehoge55** [comment]
I used a Japanese linguistic pack.
The same problem occurred.
The version is v1.9.9-1.
I'd like to know a countermeasure.

**Comment by YantaR** [comment]
Reinstall worked for me.

**Comment by hogehoge55** [comment]
I installed it again. However, it is the same symptom. 

include/il18n/ja.phar

**Comment by ntozier** [comment]
@hogehoge55 I believe when YantaR says they reinstalled... I believe that they mean they reinstalled the whole system (using the language pack) not the language pack.

Also what do you see if you go to Admin panel -> Manage -> Lists
are there are two lists with the name "Ticket Statuses"

Any errors in the PHP error log?

**Comment by hogehoge55** [comment]
Thank you very much for your correspondence.
This is an error log of PHP.
[Wed Jul 15 09:32:28 2015] [warn] [client IPaddress] mod_fcgid: stderr: PHP Fatal error:  Call to a member function all() on a non-object in /domain_name/public_html/upload/include/staff/templates/status-options.tmpl.php on line 36, referer: http://domain_name/scp/helptopics.php

**Comment by hogehoge55** [comment]
![liste](https://cloud.githubusercontent.com/assets/13326673/8688380/33f98984-2ad6-11e5-8531-385edc2c364b.JPG)

**Comment by hogehoge55** [comment]
![list2](https://cloud.githubusercontent.com/assets/13326673/8688394/64f2e36e-2ad6-11e5-8a32-e28a89f34776.JPG)

**Comment by hogehoge55** [comment]
![liste](https://cloud.githubusercontent.com/assets/13326673/8688445/3d374968-2ad7-11e5-90ea-144491a1dbd7.JPG)

**Comment by hogehoge55** [comment]
![list2](https://cloud.githubusercontent.com/assets/13326673/8688448/465da546-2ad7-11e5-987b-430708bdce4d.JPG)
![status](https://cloud.githubusercontent.com/assets/13326673/8688449/4971ee54-2ad7-11e5-8a33-84c1565caa5f.JPG)

**Comment by hogehoge55** [comment]
![ftp](https://cloud.githubusercontent.com/assets/13326673/8688485/bf9c8012-2ad7-11e5-8cc8-d69f125a671f.JPG)

**Comment by hogehoge55** [comment]
I installed only by English version.
I can choose Default Status.
![english_ver](https://cloud.githubusercontent.com/assets/13326673/8688546/63ebe810-2ad8-11e5-8c81-3a3d232d2478.JPG)

**Comment by hogehoge55** [comment]
I was able to straighten out that problem by the thing installed several times again. 
Thank you for the support.

**Comment by hossvw** [comment]
As of yesterday, I randomly could not see inside any tickets.. I can get to the list of tickets, but cannot see inside the ticket.  I upgraded from 1.9.11 to 1.9.12 hoping that it would fix it, but no luck.

I did some looking around and saw that the default ticket status is blank just like in this thread but I have not installed any language support beyond default English.

I have verified that there are indeed statuses.. I even created a new one named "Test Open" but nothing is showing up.  How can I fix this?

![screen shot 2015-12-17 at 8 17 48 am](https://cloud.githubusercontent.com/assets/16339198/11874909/d8a8843c-a496-11e5-9c6e-ebd0bb4bea68.png)

![screen shot 2015-12-17 at 8 20 08 am](https://cloud.githubusercontent.com/assets/16339198/11874962/1a4528f0-a497-11e5-8117-af83ade1b5a3.png)
